azure-pipelines.yml

```
@@ -114,7 +114,9 @@ stages:
           - Windows
           - Source_Build_Complete
           pool:
-            vmImage: windows-latest
+            name: NetCore1ESPool-Internal
+            demands: ImageOverride -equals Build.Server.Amd64.VS2019
```

Is/will there be a VS2022 image?

```suggestion
            demands: ImageOverride -equals Build.Server.Amd64.VS2022
```

---

There is a 2022 image. Why worry about VS version for this build❔

---

Turns out VS isn't used on CI at all. Arcade uses `dotnet msbuild` when global.json lacks a `tools.vs` entry, as in this repo. 2019 is fine&hellip;

---

Concern was mostly related to potential EOL of VS2019 images at some point.